### PR TITLE
LPD-35277 A collection page is not showing any results although is correctly mapped

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://liferay.atlassian.net/browse/LPD-35277

### CODE REVIEW

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java [hunk 1]
```diff
@@ -576,6 +580,75 @@ public void testDeleteObjectRelationship() throws Exception {
...
+		Bundle bundle = FrameworkUtil.getBundle(
+			ObjectRelationshipLocalServiceTest.class);
+
+		ServiceTrackerMap<String, RelatedInfoItemCollectionProvider>
+			serviceTrackerMap = ServiceTrackerMapFactory.openSingleValueMap(
+				bundle.getBundleContext(),
+				RelatedInfoItemCollectionProvider.class, "item.class.name");
+
+		Assert.assertNull(
+			null,
```
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java:616]: @marcelabc can remove this `null`

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java [hunk 2]
```diff
@@ -1295,6 +1368,9 @@ private void _testSystemObjectRelationshipOneToMany() throws Exception {
 		"R_[A-Z][0-9][A-Z][0-9]$");
 	private static ObjectDefinition _systemObjectDefinition1;
 
+	@Inject
+	private InfoItemServiceRegistry _infoItemServiceRegistry;
```
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java:1372]: @marcelabc unused variable

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java [hunk 3]
```diff
@@ -576,6 +580,75 @@ public void testDeleteObjectRelationship() throws Exception {
...
+
+		relatedInfoItemCollectionProvider = serviceTrackerMap.getService(
+			objectDefinition2.getClassName());
+
+		Assert.assertEquals(
+			objectDefinition2.getClassName(),
+			relatedInfoItemCollectionProvider.getSourceItemClassName());
+
+		_objectDefinitionLocalService.deleteObjectDefinition(objectDefinition1);
+		_objectDefinitionLocalService.deleteObjectDefinition(objectDefinition2);
```
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java:649]: @marcelabc I think you should also do `serviceTrackerMap.close()` at the end

**Review by carolmariaabb — APPROVED** [review]

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:object

**Comment by marcelabc** [comment]
I'm analyzing the CI results.

**Comment by marcelabc** [comment]
ci:report:22008158

**Comment by marcelabc** [comment]
no unique failures > relevant suite

**Comment by marcelabc** [comment]
ci:report:21983546

**Comment by marcelabc** [comment]
no unique failures > objects

**Comment by marcelabc** [comment]
ci:forward:force

**Pull request closed without merging**